editor.planx.uk/src/pages/FlowEditor/components/Flow/components/Portal.tsx

```diff
@@ -40,11 +36,6 @@ const ExternalPortal: React.FC<any> = (props) => {
           "/",
         );
         setHref(href);
-        addExternalPortal({
-          id: props.data.flowId,
-          name: data.flows_by_pk.name,
-          href,
-        });
```

Comment:
This function is required for search and will need to remain. It shouldn't be affecting the right-click functionality. For reference this was added here (via `git blame`) - https://github.com/theopensystemslab/planx-new/pull/3435